rust/cymbal/src/types/frames.rs

```diff
@@ -47,3 +48,30 @@ pub struct Frame {
     pub resolved: bool,                  // Did we manage to resolve the frame?
     pub resolve_failure: Option<String>, // If we failed to resolve the frame, why?
 }
+
+impl Frame {
+    pub fn include_in_fingerprint(&self, h: &mut Sha512) {
+        if !self.in_app {
```

Comment:
Having this check at a frame level means that if an exception has no `in_app` frames we won't include any of frames in the hash - should we not include the first frame as we did before?

Reply:
Done